src/amo/components/AddonDetail.js

```diff
@@ -47,10 +47,19 @@ export class AddonDetailBase extends React.Component {
     OverallRating: DefaultOverallRating,
   }
 
+  constructor(props) {
+    super(props);
+    this.state = { mounted: false };
+  }
+
+  componentDidMount() {
+    // Disabling react/no-did-mount-set-state because it is to prevent additional renders, but
+    // that's exactly what we want in this case (use background-image with JS).
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({ mounted: true });
```

Comment:
Yeah, the `<img>` tag is rendered on the server since we can't put `style='background-image: url("/path/to/previewURL.png")'` in the HTML served from the server.

Since long pressing on an image pops up the context menu (and it looks like disabling that isn't necessarily cross-browser and can be prevented by the user) this switches it to a `<div>` with a background image when the page loads. The background image won't open the context menu and I disabled text selection (which appears to be well supported).